docker-run.sh

```diff
@@ -21,7 +22,6 @@ replace_or_delete_in_index () {
   fi
 }
 
-replace_in_index myApiKeyXXXX123456789 $API_KEY
```

Comment:
It appears that this apiKey functionality was removed in this PR - am I missing something? If not, why was it removed?